apps/design/frontend/src/ballot_viewer.tsx

```diff
@@ -305,21 +311,51 @@ export function BallotViewer({
             ]}
           />
           <H1>View Ballot</H1>
-          <H3>Ballot Style</H3>
-          <P>{ballotStyle.id}</P>
-          <H3>Precinct</H3>
-          <P>{precinct.name}</P>
-          <H3>Page Size</H3>
-          <P>{paperSizeLabels[paperSize]}</P>
-          <H3>Timing Marks</H3>
-          <P>
-            {grid.columns} columns x {grid.rows} rows
-          </P>
-          <P>
-            <Button onPress={() => setShowGridLines(!showGridLines)} fullWidth>
-              {showGridLines ? 'Hide Grid' : 'Show Grid'}
-            </Button>
-          </P>
+          <Column style={{ gap: '1rem' }}>
+            <FormField label="Ballot Style">{ballotStyle.id}</FormField>
+            <FormField label="Precinct">{precinct.name}</FormField>
+            <FormField label="Page Size">
+              {paperSizeLabels[paperSize]}{' '}
+            </FormField>
+
+            <FormField label="Ballot Type">
+              <RadioGroup
+                options={[
+                  { value: BallotType.Standard, label: 'Precinct' },
+                  { value: BallotType.Absentee, label: 'Absentee' },
+                ]}
+                value={ballotType}
+                onChange={setBallotType}
+              />
+            </FormField>
+
+            <FormField label="Tabulation Mode">
+              <RadioGroup
+                options={[
+                  { value: 'official', label: 'Official Ballot' },
+                  { value: 'test', label: 'L&A Test Ballot' },
+                  { value: 'sample', label: 'Sample Ballot' },
+                ]}
+                value={ballotMode}
+                onChange={setBallotMode}
+              />
+            </FormField>
+
+            <FormField label="Timing Mark Grid">
+              {grid.columns} columns x {grid.rows} rows
+              <div style={{ marginTop: '0.25rem' }}>
+                <label style={{ cursor: 'pointer' }}>
+                  <input
+                    type="checkbox"
+                    checked={showGridLines}
+                    onChange={() => setShowGridLines(!showGridLines)}
+                  />{' '}
+                  Show grid lines
```

Comment:
I see that we've switched to a checkbox here, which makes a lot sense!

<img width="180" alt="spacing" src="https://github.com/votingworks/vxsuite/assets/12616928/5ad40f82-e202-4c04-8127-7fbe4d053f44">

I do think that the styling feels a bit incongruous with the radio buttons above - I've typically seen design systems style radio buttons and checkboxes similarly, at least in terms of spacing.

Not blocking to tweak the checkbox styling, just wanted to share an opinion!

Reply:
Yeah agreed, this is a bit of laziness on my part because I don't think "show grid lines" will stick around very long. Once we move to letting bubbles be placed anywhere instead of only on grid intersections, it won't be an important feature anymore. It's really only been needed to help me when building the layout algorithm.

Reply:
Got it, thanks for that context!